benchmarks/dynamo/common.py

```diff
@@ -375,17 +375,6 @@ def nothing(f):
 @functools.lru_cache(None)
 def patch_torch_manual_seed():
     """Make torch manual seed deterministic. Helps with accuracy testing."""
-
-    def deterministic_torch_manual_seed(*args, **kwargs):
```

Comment:
remove these changes too. It will be fixed in a separate PR

Reply:
regardless if it fixed in a separate PR, is it better to move the function out so it can be reuse from somewhere it needed?

Reply:
Yes, good idea. However, it's better to have 1 PR do one thing. I'll move into common util in https://github.com/pytorch/pytorch/pull/108998